consensus/reactor.go

```diff
@@ -222,6 +215,47 @@ func (conR *ConsensusReactor) Receive(chID byte, peer *p2p.Peer, msgBytes []byte
 	}
 }
 
+// Sets our private validator account for signing votes.
+func (conR *ConsensusReactor) SetPrivValidator(priv *types.PrivValidator) {
+	conR.conS.SetPrivValidator(priv)
+}
+
+// implements events.Eventable
+func (conR *ConsensusReactor) SetFireable(evsw events.Fireable) {
+	conR.evsw = evsw
+	conR.conS.SetFireable(evsw)
+}
+
+//--------------------------------------
+
+// Listens for new steps and votes,
+// broadcasting the result to peers
+func (conR *ConsensusReactor) registerEventCallbacks() {
+	// XXX: should we change SetFireable to just use EventSwitch so we don't need these assertions?
```

Comment:
Yeah maybe SetEventSwitch where possible?  Probably everywhere now since we're not using the cache, yeah?